test/xUnit/csharp/test_DefaultVisit.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Management.Automation.Language;
+using Xunit;
+
+namespace PSTests.Parallel
+{
+    public class MyCustomAstVisitor : ICustomAstVisitor2
+    {
+        public object DefaultVisit(Ast ast) => ast.GetType().Name;
+    }
+
+    public class MyAstVisotor : AstVisitor2
```

Comment:
```suggestion
    public class MyAstVisitor : AstVisitor2
```

Reply:
Also does this class need to be public?

Reply:
```suggestion
    public class MyAstVisitor2 : AstVisitor2
```

Reply:
I simply made everything public in that test. I can change them to internal.